tests/unit/spec/base/core-datalayer.js

```diff
@@ -25,6 +25,60 @@ describe('core-datalayer.js', function() {
         });
     });
 
+    describe('updateDataLayerPush', function() {
+
+        beforeAll(function() {
+            Mozilla.Analytics.updateDataLayerPush();
+        });
+
+        beforeEach(function() {
+           var link = '<a id="link" href="https://www.mozilla.org/en-US/firefox/new/">';
+            $(link).appendTo('body');
+            window.linkElement = $('#link')[0];
```

Comment:
Yes I'm talking about `linkElement`. You can do all you want above without the variable being in global scope.